docs/vdp/VDU-Commands.md

```diff
@@ -218,6 +218,8 @@ Where `char_no` is the character number to re-program, and `r1` to `r8` are the
 
 Please note that as of Console8 VDP 2.8.0 this form of the command will only work if the system font is the currently selected font.  For more information please see the [Font API](Font-API.md) documentation.
 
+Additionally, rather than re-programming the display character, there is also a function available to [remap the character to a bitmap](https://github.com/sijnstra/agon-docs/blob/main/docs/vdp/System-Commands.md#vdu-23-0-92-char-bitmapid-map-character-char-to-display-bitmapid-), allowing these font character replacements to be multi-coloured.
```

Comment:
```suggestion
Additionally, rather than re-programming the display character, there is also a function available to [remap the character to a bitmap](System-Commands.md#vdu-23-0-92-char-bitmapid-map-character-char-to-display-bitmapid-), allowing these font character replacements to be multi-coloured.
```

best not point the link to a source file in your repo :grin:

Reply:
ooooops.....